### ISSUE TITLE
[Feature]: UI Enhancements

### ISSUE THREAD

**Issue by Enigma1309** [fast4x/RiMusic#174]
### Description

1. Add an tab to Customize appearance of app in settings
-This tab should contain the Following:-
 -the default UI toggle options that toggle the official fork UI and UI that have been updated in this fork Like new player
 -A lot of people loved The Official ViMusic Player so please consider an option to toggle between old player and new one
 -An Option to move/Toggle the Search button to bottom since on small screen there's an spacing issue of search button on top.
 -Option to toggle the swipe animation, since it causes issue on old devices having slow processing
 -Swiping Down the music player causes the issue with animations and feels a bit laggy. This wasn't the case with ViMusic
 

### Suggested Solution

Basically a lot of people loved the old UI please consider and toggle option as the new player has a lot of inconsistency.

I like this project because it has an active development and a nice community ❤️

**Comment by fast4x** [comment]
Thank you for your advice

**Comment by Enigma1309** [comment]
> Thank you for your advice

Thanks for the reply i hope you will see forward on this feature ❤️

**Comment by fast4x** [comment]
Please, can you explain this "**the new player has a lot of inconsistency**" concept to me.
Thank you in advance.

**Comment by Enigma1309** [comment]
Firstly The player is not centered as it was in ViMusic (comparison below)

![Untitled](https://github.com/fast4x/RiMusic/assets/94917588/5f4aaf4c-f431-4b43-97de-982330fa3080)

Secondly It feels a lot laggy in my device (moto G4+)
The extra buttons(just above the navigation keys) feels unnecesarry and can be moved to Overflow three dots menu

Personally the ViMusic looks more aesthetic than RiMusic (just an personal preference though)

Thirdly The bottom panel has no segregation in color and feels hard to distinguish (comparison below) 
![image](https://github.com/fast4x/RiMusic/assets/94917588/7b77e959-448b-4dcd-9e1f-0ed09fcee9c5)

Moreover There's no reason to keep the Name and other element on the left and keep Centre empty (comparison below)
![image](https://github.com/fast4x/RiMusic/assets/94917588/2f3c6bac-9992-44e1-a8ea-c7c8a1d226ed)

Also the two button at top feels useless the both bring the player down and also the animation is very sketchy.
Here's a quick video comparison of the animation and lag i'm getting

https://github.com/fast4x/RiMusic/assets/94917588/35043873-20f8-4a44-8988-6d5e9e6dd430

Thanks for showing interest in my issue i hope you will come up with a fix soon

**Comment by fast4x** [comment]
Thanks for the details

**Comment by Enigma1309** [comment]
Welcome

**Comment by Enigma1309** [comment]
I didn't considered making a new issue and i'm posting it here

If possible please add support of Hyperpipe api and also ability to switch instance in piped/invidious instances
Reference:-
1) https://github.com/vfsfitvnm/ViMusic/issues/1214
2)this fork had made it possible you can see his source code:-
    https://github.com/25huizengek1/ViMusic/issues/2#issuecomment-1813044611

Also make this feature optional, with three options
use YouTube api
use Invidious api
use Piped api
use Hyperpipe api

**Comment by OrangeZXZ** [comment]
> use YouTube api
> use Invidious api
> use Piped api
> use Hyperpipe api

I like this idea because the "play with piped" and "play with invidious" takes space this should just be an setting because people would listen only on youtube or only on piped or invidious

**Comment by fast4x** [comment]
Yes it is true, you could turn the 4 voices on and off at will. Piped, Invidious, YouTube, YouTube Music.

**Comment by fast4x** [comment]
Api for now isn't supported

**Comment by Enigma1309** [comment]
> Api for now isn't supported

okay i respect your opinion and i hope to see it in coming future

**Comment by fast4x** [comment]
Sure
